README.md

```diff
@@ -82,22 +82,58 @@ Navigate to http://localhost:8080/opsui, you should see the default OPSUI system
 
 To contribute a patch, follow these instructions (note that installing
 [Hub](http://hub.github.com) is not strictly required, but is recommended).
+```
+Download and install hub.github.com
+```
+1.File JIRA issue for your fix at https://issues.apache.org/jira/browse/OODT.
+you will get issue id OODT-xxx where xxx is the issue ID.
+
+2.[Fork the repo](http://help.github.com/fork-a-repo) on which you're working, clone your forked repo to your local computer, and set up the upstream remote:
+```
+git clone https://github.com/YourGitHubUserName/oodt.giy
```

Comment:
A small typo here
```suggestion
git clone https://github.com/YourGitHubUserName/oodt.git
```
Also let's make the username part **<username>** so that it is more clear.
```suggestion
git clone https://github.com/<YourGitHubUserName>/oodt.giy
```

Reply:
Fixed as suggested.